### PR TITLE
ODF template  update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

File: config-templates/redhat/odf-local/4.14/custom-parameters.json [hunk 1]
```diff
@@ -30,6 +30,14 @@
     "required": "true",
     "mutable": "true"
   },
+  {
+    "description": "A comma separated list of the worker pool names where you want to deploy ODF. Either specify workerpool or workernodes to deploy ODF, if not specified ODF will deploy on all nodes",
+    "displayname": "Worker pool names",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.14/custom-parameters.json]: `Worker pool names` --> `Worker pools name` or something similar

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,56 +3,58 @@
         "4.14": {
             "status": "supported",
             "latest-revision": {
-                "revision": "3",
-                "release-date": "07 March 2024",
-                "cves": "CVE-2019-13224, CVE-2019-16163, CVE-2019-19012, CVE-2019-19203, CVE-2019-19204",
-                "golang": "1.21.7",
+                "revision": "4",
+                "release-date": "19 April 2024",
+                "cves": "CVE-2023-28322,CVE-2023-38546,CVE-2023-46218",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: please add spaces between CVEs

File: config-templates/redhat/odf-local/changelog.json [hunk 3]
```diff
@@ -3,56 +3,58 @@
...
     },
     {
         "4.13": {
-            "status": "supported",
-            "latest-revision": {
-                "revision": "8",
-                "release-date": "07 March 2024",
-                "cves": "CVE-2019-13224, CVE-2019-16163, CVE-2019-19012, CVE-2019-19203, CVE-2019-19204",
-                "golang": "1.21.7",
+            "status": "supported","latest-revision": {
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Fix this line

File: config-templates/redhat/odf-local/changelog.json [hunk 4]
```diff
@@ -61,56 +63,33 @@
...
-                "cves": "CVE-2021-35937, CVE-2021-35938, CVE-2021-35939, CVE-2023-48795, CVE-2024-0553",
-                "golang": "1.21.7",
-                "ubi": "",
-                "new-features": [
-                ],
-                "fixes": [
-                    "Replaced UBI image with scratch image"
-                ]
-            }
+            "status": "unsupported"
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json:93]: update status in metadata.json as well

File: config-templates/redhat/odf-local/4.14/custom-parameters.json [hunk 5]
```diff
@@ -30,6 +30,14 @@
...
+    "name": "worker-pools",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.14/custom-parameters.json:36]: Are `worker-nodes` & `worker-pools` mutually exclusive parameter?

File: config-templates/redhat/odf-local/4.14/custom-parameters.json [hunk 6]
```diff
@@ -30,6 +30,14 @@
     "required": "true",
     "mutable": "true"
   },
+  {
+    "description": "A comma separated list of the worker pool names where you want to deploy ODF. Either specify workerpool or workernodes to deploy ODF, if not specified ODF will deploy on all nodes",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.14/custom-parameters.json]: Please get this reviewed by Doc team.

File: config-templates/redhat/odf-remote/4.14/custom-parameters.json [hunk 7]
```diff
@@ -30,6 +30,14 @@
     "required": "true",
     "mutable": "true"
   },
+  {
+    "description": "A comma separated list of the worker pool names where you want to deploy ODF. Either specify workerpool or workernodes to deploy ODF, if not specified ODF will deploy on all nodes",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/4.14/custom-parameters.json]: Same comments as per odf-remote template

File: config-templates/redhat/odf-remote/changelog.json [hunk 8]
```diff
@@ -63,58 +66,34 @@
...
-                "cves": "CVE-2021-35937, CVE-2021-35938, CVE-2021-35939, CVE-2023-48795, CVE-2024-0553",
-                "golang": "1.21.7",
-                "ubi": "",
-                "new-features": [
-                ],
-                "fixes": [
-                    "Replaced UBI image with scratch image"
-                ]
-            }
+            "status": "unsupported"
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json:96]: update status in metadata.json as well

**Review by mssachan — CHANGES_REQUESTED** [review]
Please fix the review comments.

**Comment by gayathrimenath-ibm** [comment]
@derekpoindexter Could you please verify the new parameter description

**Comment by mssachan** [comment]
@derekpoindexter The newly added param `worker-pools` & existing param `worker-nodes` are mutually exclusive. So I think we need to mention this in description of both params. Can you please review description of both parameters and suggest the changes.

File: config-templates/redhat/odf-local/4.14/custom-parameters.json [hunk 9]
```diff
@@ -30,6 +30,14 @@
     "required": "true",
     "mutable": "true"
   },
+  {
+    "description": "A comma separated list of the workerpools names where you want to deploy ODF. Either specify workerpools or workernodes to deploy ODF. if not specified any of them, ODF will deploy on all nodes",
```
Comment (derekpoindexter) [inline config-templates/redhat/odf-local/4.14/custom-parameters.json]:
```suggestion
    "description": "Specify a comma separated list of either the names or the IDs of the worker pools you want to use for ODF. If you want to use all the worker nodes in your cluster for ODF, then don't specify this option. ",
```

Comment (gayathrimenath-ibm) [reply config-templates/redhat/odf-local/4.14/custom-parameters.json]: @derekpoindexter Customer can select any of the below options
1. workerpool/list of workerpools -> ODF will install on all nodes under specified workerpools
2. list of worker nodes - ODF will install on specified workernodes
3. not specified any workerpool/workernode - ODF will install on all nodes. 

The second condition is not added in above description. I think we have to specify those also to understand better

**Review by mssachan — APPROVED** [review]
lgtm

**Pull request merged**